src/components/PriceInputRow.vue

```diff
@@ -59,6 +61,18 @@ export default {
     setCurrencyData(currency) {
       this.priceForm.currency = currency
     },
+    fixComma(input) {
+      return input.replace(/,/g, '.');
+    },
+  },
+  computed: {
+    priceRules() {
+      return [
+        value => !!value && !!value.trim() || 'Price is required',
+        value => !isNaN(value) || 'Input a number',
+        value => Number(value) >= 0 || 'Price may not be negative',
```

Comment:
add a rule with a maximum of 2 decimals ?

Reply:
fyi i added a PR with 2 extra tests in the backend : https://github.com/openfoodfacts/open-prices/pull/279

Reply:
I added the rule of (max) 2 decimals, as it was simple and it's important to catch if someone uses a thousands separator "1.000".